src/app/server/Server.h

```diff
@@ -303,16 +303,35 @@ class Server
             return CHIP_NO_ERROR;
         };
 
-        void OnFabricDeletedFromStorage(CompressedFabricId compressedId, FabricIndex fabricIndex) override
+        void OnFabricDeletedFromStorage(FabricTable & fabricTable, FabricIndex fabricIndex) override
         {
-            (void) compressedId;
+            (void) fabricTable;
```

Comment:
Generally speaking: why is the fabricTable argument there at all in `OnFabricDeletedFromStorage`?  The fabric index can't be used to index into the table (because that fabric has been removed).  What else would the table be used for?

Reply:
It could be removed. I wanted symmetry. In theory you could look at the rest of the fabric table, while at the same time know what fabric got removed to pass it to your cleaners. I can remove fabricTable

Reply:
I don't have a strong opinion here, fwiw.  Keeping it is fine too....

Reply:
Will keep as is